src/test/java/hu/oe/nik/szfmv/automatedcar/model/ShapeTest.java

```diff
@@ -0,0 +1,86 @@
+package hu.oe.nik.szfmv.automatedcar.model;
+
+import hu.oe.nik.szfmv.automatedcar.AutomatedCar;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class ShapeTest {
```

Comment:
A Sign-nál és a Tree-nél legalább egy Z indexet checkoljunk légyszi (főleg demo előtt), az csak plusz, ha a poligonokat is ellenőrizzük :D